site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
+| Image Quality        | Use this parameter to specify the compression level for uploaded images and to improve the speed of loading for high-resolution datasets. <br> Values range from `5` (highly compressed images) to `100` (not compressed images).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
The option confuses many users. There two types of data in CVAT: original and compressed. The original images are used then you export a dataset, or automatically annotate data. The compressed images are used only for annotations. It will reduce the traffic between the server and clients. I will recommend adjusting the parameter only if images have very small objects which are not visible with the default compression level.